src/stacks-hierarchy/Root_ParkingViolationsReporting/Root_ParkingViolationsQr.tsx

```diff
@@ -113,7 +113,10 @@ export const Root_ParkingViolationsQr = ({
   const selectProvider = (qrScanFailed?: boolean) => {
     openBottomSheet(() => (
       <SelectProviderBottomSheet
-        providers={providers}
+        providers={[
+          ...providers,
+          {name: t(ParkingViolationTexts.selectProvider.unknownProvider)},
+        ]}
```

Comment:
I think this should be done with `useMemo`. If you have a hook which has `providers` in the dependency list this would fire on every render now, as the array is recreated on every render.